batch/batch/front_end/templates/table_search.html

```diff
@@ -49,7 +49,7 @@
       <form method="GET" action="{{ endpoint }}">
         <div id="{{ component_name }}-inner">
           <div>
-            <textarea id="{{ component_name }}-input-box" autofocus name="q">{% if q %}{{ q }}{% endif %}</textarea>
+            <textarea id="{{ component_name }}-input-box" spellcheck="off" autocorrect="off" autofocus name="q">{% if q %}{{ q }}{% endif %}</textarea>
```

Comment:
oop, sorry, i should have been more explicit; it's `off` for `autocorrect`, but still `false` for `spellcheck`.

```suggestion
            <textarea id="{{ component_name }}-input-box" spellcheck="false" autocorrect="off" autofocus name="q">{% if q %}{{ q }}{% endif %}</textarea>
```

otherwise LGTM though!

also lol, kinda goofy that the double-space period setting doesn't respect the `autocorrect` attribute 🙃